src/scripts.js

```diff
@@ -111,3 +165,30 @@ const findDestination = (destinationName) => {
   return destinationID.id;
 }
 
+const verifyLogin = (data) => {
+  console.log(usernameInput.value.slice(8))
+  if (passwordInput.value === 'travel') {
+    show(tripCardContainer);
+    show(header);
+    show(tripButtons);
+    hide(loginBox);
+  } else {
+    wrongPwdField.innerText = 'Incorrect password! Please try again.'
+  }
+  const travelerID = usernameInput.value.slice(8);
+  fetchData(travelerID)
```

Comment:
What do you think about moving lines 178-179 inside of the if statement? Shouldn't your fetchData(travelerID) call only occur if the password is correct?